espanol/noticias/burla-periodista/leones/leones.md

```diff
@@ -1,3 +1,9 @@
 El periodista queda rodeado por 4 expectaculares fieras 
 que podrían destrozrlo tan solo al escuchar a Beatito 
 chasquir sus dedos.
+
+
+Y después:
+
+
+[Se abalanzó](../leones/lanzaronPeriodista.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.